requirements/dev.txt

```diff
@@ -2055,9 +2042,9 @@ rpds-py==0.22.3 \
     #   -r requirements/prod.txt
     #   jsonschema
     #   referencing
-rq==2.0.0 \
-    --hash=sha256:76d2a4a27f8fd5c4cfa200cd442efe3c1fd73525c676af06f07fcc0b81bdb70d \
-    --hash=sha256:a3a767876675dcc42683bac1869494c5020ba7fcf5c026d1f6d36a8ab98573a6
+rq==2.1.0 \
+    --hash=sha256:3c6892c6ca848e5fb47c1875399a66f13656bf0e123bf725d9aa9a12718e2fdf \
+    --hash=sha256:764585b6cab69ef1412f4aee523347e5aa7ece3ca175c118b1d92223dd8c2826
```

Comment:
Not seeing anything major in the [CHANGES](https://github.com/rq/rq/blob/master/CHANGES.md) file